packages/dnb-eufemia/src/extensions/forms/Iterate/PushContainer/PushContainer.tsx

```diff
@@ -17,18 +17,43 @@ import OpenButton from './OpenButton'
 import { Flex, HeightAnimation } from '../../../../components'
 import { OnCommit, Path } from '../../types'
 import { SpacingProps } from '../../../../shared/types'
-import { useArrayLimit, useSwitchContainerMode } from '../hooks'
+import {
+  useArrayLimit,
+  useSwitchContainerMode,
+  useItemPath,
+} from '../hooks'
 import Toolbar from '../Toolbar'
 import { useTranslation } from '../../hooks'
 import { ArrayItemAreaProps } from '../Array/ArrayItemArea'
 import { clearedData } from '../../DataContext/Provider'
 
-export type Props = {
+/**
+ * Deprecated, as it is supported by all major browsers and Node.js >=v18
+ * So it's a question of time, when we will remove this polyfill
+ */
+import structuredClone from '@ungap/structured-clone'
+
+type OnlyPath = {
   /**
    * The path to the array to add the new item to.
    */
   path: Path
 
+  /** The sub path to the array to add the new item to. */
+  itemPath?: Path
+}
+
+type OnlyItemPath = {
```

Comment:
```suggestion
type OnlyItemPath = {
```

If the type is "only item path", does it still make sense to have the path property here as well? As the type says "only item path"?

Or perhaps it could be named `OnlyItemPathRequired` or so?
Same for `OnlyPath`, perhaps `OnlyPathRequired`?